### DISCUSSION TITLE
bug in termux

### DISCUSSION THREAD

**Discussion by fazle-rabbi-dev** [officialputuid/toolsig#55] [General]
![Screenshot_20220113-143647](https://user-images.githubusercontent.com/71178740/149295494-c2be6e0b-0018-4742-a814-3f20ae621d5f.png)

**Comment by refx3** [comment]
downgrade to npm version 6 using:
`npm install -g npm@6`
then run `npm i` again.

**Comment by officialputuid** [comment]
1. Make sure you download termux on fdroid, because the update on playstore has been discontinued.

2. use nodejs-lts (read tutorial please *)
* https://github.com/officialputuid/toolsig#tutorial-on-termux-android
